ui/components/MesheryPerformance/PerformanceProfiles.js

```diff
@@ -215,32 +245,20 @@ function PerformanceProfile({ updateProgress, enqueueSnackbar, closeSnackbar })
                 onClick={() => setProfileForModal({})}
               >
                 <AddIcon />
-              Add Performance Profile
+                Add Performance Profile
               </Button>
             </div>
           </Paper>
         ) : (
-          <div style={{ width : "fit-content", margin : "4rem auto 0" }}>
-            <Button
-              aria-label="Add Performance Profile"
-              variant="contained"
-              color="primary"
-              size="large"
-              // @ts-ignore
-              onClick={() => setProfileForModal({})}
-            >
-              <AddIcon />
-            Add Performance Profile
-            </Button>
-          </div>
+          console.log("yay")

```

Comment:
Can we do the same thing here too,

Reply:
@Abhishek-kumar09 I think you reviewed the previous commit😅.